src/index.js

```diff
@@ -0,0 +1,54 @@
+const cluster = require ("cluster");
+const  { cpus } = require ("os");
```

Comment:
Codacy has a fix for the issue: [Replace `··{·cpus·}·=·require·` with `·{·cpus·}·=·require`](https://app.codacy.com/gh/armerillo/ql-task/pullRequest?prid=9973834)
```suggestion
const { cpus } = require("os");
```